browser/index.html

```diff
@@ -1,45 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <title>Title</title>
-  <script
-    src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.14.0/babel.min.js"
-    data-presets="es2015,react,stage1"
-  ></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.3.2/react.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.3.2/react-dom.js"></script>
-  <script src="../dist/umd/semantic-ui-react.min.js"></script>
-  <link
-    rel="stylesheet"
-    href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.2/semantic.min.css"
-  >
-</head>
-<body>
-  <script type="text/babel">
-    const { Container, Divider, Dropdown, Header, Message } = semanticUIReact
-
-    const options = [
-      { value: 'all', text: 'All' },
-      { value: 'articles', text: 'Articles' },
-      { value: 'products', text: 'Products' },
-    ]
-
-    const App = () => (
-      <Container>
-        <Divider hidden section />
-        <Header as='h1'>Browser Build</Header>
-        <Message info icon='info' content='This app tests the browser build.' />
-
-        <Dropdown selection options={options} placeholder='Choose one...' />
-      </Container>
-    )
-
-    const mountNode = document.createElement('div')
-    document.body.appendChild(mountNode)
-
-    ReactDOM.render(<App />, mountNode)
-  </script>
-</body>
-</html>
```

Comment:
Cruft file

Reply:
This file allowed manual testing to ensure the UMD build worked.  It was originally created to verify that the codepen would work as codepen used the UMD bundle.  Since we now use codesandbox and it doesn't need a UMD, it is probably safe to remove.

In fact, I wonder if we anyone actually uses the UMD bundle at all.  It would simplify and speed things up if we removed it.

Reply:
I'm think that we will drop UMD build in near future :+1: